eclair-core/src/main/scala/fr/acinq/eclair/io/PendingChannelsRateLimiter.scala

```diff
@@ -30,6 +30,7 @@ object PendingChannelsRateLimiter {
   private case class WrappedGetNodeResponse(temporaryChannelId: ByteVector32, response: Router.GetNodeResponse, replyTo: Option[ActorRef[Response]]) extends Command
   private case class ReplaceChannelId(remoteNodeId: PublicKey, temporaryChannelId: ByteVector32, channelId: ByteVector32) extends Command
   private case class RemoveChannelId(remoteNodeId: PublicKey, channelId: ByteVector32) extends Command
+  private[io] case class OpenChannelRequests(replyTo: ActorRef[Int], publicPeers: Boolean) extends Command
```

Comment:
nit: it's easier to understand what this does if the name is an action:
```suggestion
  private[io] case class CountOpenChannelRequests(replyTo: ActorRef[Int], publicPeers: Boolean) extends Command
```

Reply:
Good point; atleast I avoided the G word though! fixed in [1f1221f](https://github.com/ACINQ/eclair/pull/2606/commits/1f1221fc6a3231bc70be301f2b070c3f775abec0).